index.html

```diff
@@ -297,8 +296,59 @@ <h2>Contact me</h2>
         </section>
     </section>
 
-    <script src="./JS/"></script>
-    <script src="./JS/"></script>
+    <!-- Modal -->
+    <section class="modalContainer" id="modalContainers">
+      <div class="modalHeader">
+          <div class="modalNav">
+            <h3 class="card-title" id="modalTitle"></h3>
+            <div class="modalNavLi" id="close-btn">&times;
+
+            </div>
+          </div>
+          <ul class="canopy" id="canopy">
+            <!-- <li class="canopyLi1">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li> -->
+          </ul>
+          <div class=" modalImg" id="modal-Img">
+            <!-- <img src="" alt="Card Image" /> -->
+          </div>        
+        <div class="modalFooter">
+          <p id="modalDes" >
+          </p>
+        <div class="modal-info">
+          <ul id="modalLang" >
+            <!-- <li>html</li>
+            <li>css</li>
+            <li>Javascript</li> -->
+          </ul>
+
+         <hr>
```

Comment:
Please change the color of the hr element to match the figma design 👨‍🎨
<img width="798" alt="Screenshot 2023-02-03 at 11 26 04 PM" src="https://user-images.githubusercontent.com/25887253/216705875-fe0ff961-a82e-4144-ab91-25524e3f5623.png">